### PR TITLE
refactor npu_model

### PR DESCRIPTION
**Pull request by lzivan**
## Description

- replace enable_mp parameter as optimize_model
- customize default value of inter_pp and intra_pp by model type

### CODE REVIEW

**Comment by plusbang** [comment]
Please fix code style and add descriptions about parameter of `from_pretrained`

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen2.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
         trust_remote_code=True,
         attn_implementation="eager",
         load_in_low_bit="sym_int4",
-        enable_mp=True,
+        optimize_modelin=True,
```
Comment (plusbang) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen2.py:58]: Fix this typo also.

**Comment by jason-dai** [comment]
Let's fix the typos and merge this PR @plusbang @lzivan

**Review by plusbang — APPROVED** [review]

**Comment by plusbang** [comment]
> Let's fix the typos and merge this PR @plusbang @lzivan

Have merged :)

**Pull request merged**